### PR TITLE
Update descriptions for the docker volume sub-command, such as  create，list and inspect

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
Update descriptions for the docker volume sub-command, such as  create，list and inspect

**- How I did it**
**- How to verify it**

**- Description for the changelog**

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: cli/command/volume/cmd.go [hunk 1]
```diff
@@ -36,13 +36,13 @@ volume is a specially-designated directory that by-passes storage driver
...
 Data volumes persist data independent of a container's life cycle. When you
-delete a container, the Engine daemon does not delete any data volumes. You can
+delete a container, the Docker daemon does not delete any data volumes. You can
 share volumes across multiple containers. Moreover, you can share data volumes
 with other computing resources in your system.
 
 To see help for a subcommand, use:
 
-    docker volume CMD help
+    docker volume COMMAND --help
```
Comment (mstanleyjones) [inline cli/command/volume/cmd.go]: Is this syntax standard across all the files?

File: cli/command/volume/inspect.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ func runInspect(dockerCli *command.DockerCli, opts inspectOptions) error {
 }
 
 var inspectDescription = `
-Returns information about one or more volumes. By default, this command renders
+Return information about one or more volumes. By default, this command renders
```
Comment (mstanleyjones) [inline cli/command/volume/inspect.go]: You're not returning, the command is. I think it needs to stay how it was.

File: cli/command/volume/create.go [hunk 3]
```diff
@@ -72,7 +72,7 @@ func runCreate(dockerCli *command.DockerCli, opts createOptions) error {
 }
 
 var createDescription = `
-Creates a new volume that containers can consume and store data in. If a name
+Create a new volume that containers can consume and store data in. If a name
```
Comment (mstanleyjones) [inline cli/command/volume/create.go]: You're not creating the volume, the command is. I think it needs to stay how it was.

File: cli/command/volume/list.go [hunk 4]
```diff
@@ -76,7 +76,7 @@ func runList(dockerCli *command.DockerCli, opts listOptions) error {
 
 var listDescription = `
 
-Lists all the volumes Docker knows about. You can filter using the **-f** or
+List all the volumes Docker knows about. You can filter using the **-f** or
```
Comment (mstanleyjones) [inline cli/command/volume/list.go]: I'd keep it how it was, but change 'knows about' to 'manages'.

**Review by mdlinville — CHANGES_REQUESTED** [review]

File: cli/command/volume/cmd.go (hunk 1 above)
Comment (yuexiao-wang) [inline cli/command/volume/cmd.go]: I can not find it which is a long  description for CLI command.  I modified it  which referring to the help as follows: 
$ docker volume --help

Usage:	docker volume COMMAND

Manage Docker volumes

Options:
      --help   Print usage

Commands:
  create      Create a volume
  inspect     Display detailed information on one or more volumes
  ls          List volumes
  rm          Remove one or more volumes

Run 'docker volume COMMAND --help' for more information on a command.

File: cli/command/volume/inspect.go (hunk 2 above)
Comment (yuexiao-wang) [inline cli/command/volume/inspect.go]: Got it and I will update the PR

File: cli/command/volume/create.go (hunk 3 above)
Comment (yuexiao-wang) [inline cli/command/volume/create.go]: Got it and I will update the PR

File: cli/command/volume/list.go (hunk 4 above)
Comment (yuexiao-wang) [inline cli/command/volume/list.go]: Got it and I will apply it to this PR.
Thanks

**Comment by yuexiao-wang** [comment]
@mstanleyjones PTAL
Done！

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

File: docs/reference/commandline/volume_create.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ Options:
   -o, --opt value       Set driver specific options (default map[])
 ```
 
-Creates a new volume that containers can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
+Create a new volume that containers can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
```
Comment (thaJeztah) [inline docs/reference/commandline/volume_create.md]: Wait: @yuexiao-wang did you just push again? Looks like this was changed back again

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/volume_inspect.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ Options:
       --help            Print usage
 ```
 
-Returns information about a volume. By default, this command renders all results
+Return information about a volume. By default, this command renders all results
```
Comment (thaJeztah) [inline docs/reference/commandline/volume_inspect.md]: This as well

Comment (yuexiao-wang) [inline docs/reference/commandline/volume_inspect.md]: Sorry，it is  careless operation. I want to restart CI when experimental is failed.  I checked it and update the PR.

**Comment by yuexiao-wang** [comment]
@thaJeztah PTAL

File: docs/reference/commandline/volume_create.md (hunk 5 above)
Comment (yuexiao-wang) [inline docs/reference/commandline/volume_create.md]: Updated

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**